src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java

```diff
@@ -571,13 +572,11 @@ private void calcAdditionalInformation(final OffenderBooking booking) {
         final var rawData = jdbcTemplate.query(
                 InmateRepositorySql.GET_RECATEGORISE.getSql(),
                 createParams("agencyId", agencyId,
-                        "assessStatus", Set.of("A", "P"),
```

Comment:
If I'm taking out this filter in the SQL, do we think we should add in the filter on the final step removeNonStandardCategoryRecords() to only return assessments with a status of 'A' or 'P' 🤔

Reply:
yes, we should return the 'current' cat, not an inactive one

Reply:
Also does assessStatus need removing from the SQL

Reply:
It's what I did on the last PR, if memory serves right I think it was because anything but the recent one was marked as 'I' (Inactive), so wouldn't be able to look at the previous assessments for an offender if we left it in (might be wrong though!)

Suppose the question is do we want to return the latest assessment if it's Inactive?

Reply:
Yes what i meant was that its fine to remove assessStatus from the SQL, and we should return the 'current' cat (i.e. the A or P), not an inactive one, _from the api_.